### PR TITLE
Test unique deploy

### PR DESCRIPTION
**Pull request by alareix**
please review changes to signer mister Yulian

### CODE REVIEW

File: sign/views.py [hunk 1]
```diff
@@ -25,13 +28,24 @@ def sign_view(request):
...
-        raw_hex_tx = signed_tx.rawTransaction.hex()
-        return JsonResponse({'signed_tx': raw_hex_tx})
-    elif account.network_type == NetworkType.BINANCE_CHAIN:
-        raise PermissionDenied
+        if account.network_type == NetworkType.ETHEREUM_LIKE:
+            signed_tx = Web3().eth.account.sign_transaction(tx_params, account.private_key)
+            raw_hex_tx = signed_tx.rawTransaction.hex()
+            return JsonResponse({'signed_tx': raw_hex_tx})
+        elif account.network_type == NetworkType.BINANCE_CHAIN:
+            raise PermissionDenied
```
Comment (qwdqwqdwqd) [inline sign/views.py:48]: refactor ^=^

**Review by qwdqwqdwqd — CHANGES_REQUESTED** [review]

File: sign/views.py [hunk 2]
```diff
@@ -19,18 +30,23 @@ def sign_view(request):
         raise PermissionDenied
 
     tx_params = request.data
-    try:
+    if tx_params.get('to'):
```
Comment (yulian-khalitov) [inline sign/views.py:33]: get('to', None)

File: sign/views.py [hunk 3]
```diff
@@ -19,18 +30,23 @@ def sign_view(request):
...
+        print('No destination address provided')
 
     try:
         account = BlockchainAccount.objects.get(address=tx_params.pop('from'))
     except BlockchainAccount.DoesNotExist:
         raise PermissionDenied
 
     if account.network_type == NetworkType.ETHEREUM_LIKE:
-        signed_tx = Web3().eth.account.sign_transaction(tx_params, account.private_key)
+        if tx_params.get('contractId'):
```
Comment (yulian-khalitov) [inline sign/views.py:44]: get('child_id', None)

**Review by yulian-khalitov — CHANGES_REQUESTED** [review]

File: sign/views.py (hunk 2 above)
Comment (orionoiro) [reply sign/views.py:33]: @_@

**Review by qwdqwqdwqd — APPROVED** [review]

**Review by qwdqwqdwqd — APPROVED** [review]

**Pull request merged**